core/cap-0021.md

```diff
@@ -83,9 +89,7 @@ struct AccountEntryExtensionV3
 {
     Liabilities liabilities;
 
-    uint32 numSponsored;
-    uint32 numSponsoring;
-    SponsorshipDescriptor signerSponsoringIDs<MAX_SIGNERS>;
+    AccountEntryExtensionV2 *sponsorInfo;
```

Comment:
Nice. This will make it easier to keep the existing logic and invariants for sponsorship.